tag.go

```diff
@@ -59,35 +93,45 @@ func (t Tag) String() string {
 	}
 	if len(t.Body) > 0 {
 		bb.WriteString(">")
+		// Function for opening tag END - if body
 		for _, b := range t.Body {
-			switch tb := b.(type) {
-			case htmler:
-				bb.Write([]byte(tb.HTML()))
-			case fmt.Stringer:
-				bb.WriteString(tb.String())
-			case interfacer:
-				val := tb.Interface()
-				if tb.Interface() == nil {
-					val = ""
-				}
-
-				bb.WriteString(fmt.Sprint(val))
-			default:
-				bb.WriteString(fmt.Sprint(tb))
-			}
+			parseTagEmbed(b, bb)
 		}
+
 		bb.WriteString("</")
 		bb.WriteString(t.Name)
 		bb.WriteString(">")
+
+		if len(t.AfterTag) > 0 {
+			for _, at := range t.AfterTag {
+				parseTagEmbed(at, bb)
+			}
+		}
+
 		return bb.String()
 	}
 	if !strings.Contains(voidTags, " "+t.Name+" ") {
+		// Function for opening tag END - if no body
 		bb.WriteString("></")
 		bb.WriteString(t.Name)
 		bb.WriteString(">")
+
+		if len(t.AfterTag) > 0 {
```

Comment:
You don't need this `if` statement. The `range` will only run if there anything in the slice.